test/input_gen/genModelTests_v2.py

```diff
@@ -32,4 +69,10 @@ def forward(self, inputs, labels):
         name="reduce_mean_last",
     )
 
-    # inspect_file("lstm_single.nnmodelgolden")
```

Comment:
Could you leave this at the last line for the reference?

Reply:
done.